manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ZebraTechnologiesCorporation.ZebraDesigner
+PackageVersion: 2.5.0.9420
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Designer/zebradesigner-v250-9420.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: FAE8DD644EB426C0DC394A50FB1BC1E238451180BFB2A4E13B82ED87DDC77F74

```

Comment:
```suggestion
  InstallerSha256: FAE8DD644EB426C0DC394A50FB1BC1E238451180BFB2A4E13B82ED87DDC77F74
  InstallerSwitches:
    Silent: /s
```

Reply:
This package uses the InstallAware Wizard, which uses the `/s` switch for [silent install](https://www.installaware.com/forums/viewtopic.php?t=4442#:~:text=To%20do%20this%20you%20can%20go%20to%20Start,%3Csetup%3E%20%2Fs%20this%20will%20run%20the%20installer%20silently.). I do not know if there is a silent-with-progress switch.